manifests/init.pp

```diff
@@ -21,4 +22,11 @@
 
   if $include_abalone { include learning::abalone }
 
+  if $autosign_value {
```

Comment:
This seems reasonable for this specific purpose. If you intend on extending this use to auto signing specific host patterns, you might use an array and a template like

```
<% @autosign_hosts.each do |pattern| -%>
pattern
<% end -%>
```